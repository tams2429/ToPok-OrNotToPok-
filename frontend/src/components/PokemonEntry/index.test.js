import { render } from '@testing-library/react'
import { PokemonEntry } from './index'

describe('Pokemon Entry Card Component', () => {

  const initialProps = {
    name: 'mewtwo',
    description: 't wast did create by a scientist after years of horrific gene splicing and dna engineering experiments.',
    sprite: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/150.png'
  }

  it('should render properly and match snapshot', () => {
    const pokemonEntryComponent = render(<PokemonEntry {...initialProps}/>)
    expect(pokemonEntryComponent.container).toMatchSnapshot()
  })
})