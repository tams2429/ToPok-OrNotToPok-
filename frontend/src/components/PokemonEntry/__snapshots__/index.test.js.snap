// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemon Entry Card Component should render properly and match snapshot 1`] = `
<div>
  <section
    class="pokemon-entry-container"
  >
    <img
      alt="Searched pokemon sprite appearance"
      class="pokemon-img"
      src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/150.png"
    />
    <h2
      class="pokemon-name"
    >
      mewtwo
    </h2>
    <p
      class="pokemon-description"
    >
      t wast did create by a scientist after years of horrific gene splicing and dna engineering experiments.
    </p>
  </section>
</div>
`;
